e2e-tests/test/transactions.js

```diff
@@ -1,70 +1,94 @@
 const { assert } = require('console');
 const NodeClient = require('../node-client');
 const { nodes } = require('./../config.json');
+const { UInt, TypeRegistry } = require("@polkadot/types");
+const U128 = UInt.with(128, "U128");
+const U64 = UInt.with(64, "U64");
+
 describe('Transactions', function () {
     this.timeout(10_000);
 
     let node1, node2, node3;
+    let node1Obj = nodes[0];
+    let node2Obj = nodes[1];
+    let node3Obj = nodes[2];
+    let node4Obj = nodes[3];
```

Comment:
Initially, I read `node10 bj` and `node20 bj`... maybe we can rename them to simply `node1`, `node2` etc and the clients to `node1Client` `node2Client`